command/execute.go

```diff
@@ -100,6 +127,10 @@ func (c *ExecuteCommand) Run(args []string) int {
 		return 1
 	}
 
+	if args.UseProtobuf {
+		server.UseProto = true
```

Comment:
Do the builtin plugins always use protobufs if supported? How do we toggle it off for built ins?

Reply:
So the main `Packer` process is responsible for that, internal components are still executed much like any binary through `packer execute`, so if plugins support protobuf, we'll add the flag to the command, and the internal components will run as a server with protobuf.

Also I believe we can do `server.UseProto = args.UseProtobuf` instead of the `if` here :)

Reply:
Okay understood. Yeah these plugins are also not plugin.Set right so we have to set `server.UseProto = args.UseProtobuf` got it now.

Reply:
Yeah exactly, `plugin.Set` is used only for plugins, as they register their components, and we can use the rest of the predefined commands for them.
Since this is internal, there's no concept of `describe` for example (not even sure the command's called `start` for an internal component), so `Set` is not used, but instead we directly create the server with the component requested by name.